src/load.test.ts

```diff
@@ -71,7 +75,7 @@ describe('load behaves as expected', () => {
     process.env = OLD_ENV
   })
 
-  it('returns the memoized state if didLoad is true', () => {
+  it('returns the memoized state loaded config is not nil', () => {
```

Comment:
grammar is off here. are you missing an `if`?

Reply:
fixed